client/src/components/ContentForm.js

```diff
@@ -0,0 +1,108 @@
+const { useState } = require("react")
+
+const ContentForm = () => {
+    const [name, setName] = useState('')
+    const [url, setURL] = useState('')
+    const [workout_type, setType] = useState('')
+    const [duration, setDuration] = useState('')
+    const [difficulty_level, setDifficulty] = useState('Beginner')
+    const [thumbnail_url, setTurl] = useState('')
+    const [description, setDescription] = useState('')
```

Comment:
consider putting all of form attributes into one state, something like this:
const [formData, setFormData] = useState({
name: '',
url: '',
workout_type: '',
duration: '',
difficulty_level: 'Beginner',
thumbnail_url: '',
description: ''
});